src/Symfony/Maker/Resources/skeleton/StateProvider.tpl.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+echo "<?php\n"; ?>
+
+namespace <?php echo $namespace; ?>;
+
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\State\ProviderInterface;
+
+class <?php echo $class_name; ?> implements ProviderInterface
+{
+    public function provide(Operation $operation, array $uriVariables = [], array $context = [])<?php if (\PHP_VERSION_ID >= 80000) { ?>: object|array|null<?php } ?>
+
```

Comment:
```suggestion
```

Reply:
because of the `if` at the end of the line, this break row is mandatory

you can look at `CustomStateProviderPhp{7,8}.php` in tests fixtures, the break row is not present

Reply:
Understood, thanks!